lua/telescope/builtin/__git.lua

```diff
@@ -395,7 +395,11 @@ local set_opts_cwd = function(opts)
     local in_bare = utils.get_os_command_output({ "git", "rev-parse", "--is-bare-repository" }, opts.cwd)
 
     if in_worktree[1] ~= "true" and in_bare[1] ~= "true" then
-      error(opts.cwd .. " is not a git directory")
+      utils.notify("builtin.git", {
+        msg = opts.cwd .. " is not a git directory",
+        level = "ERROR",
+      })
+      return false
```

Comment:
i prefer not to use vim.notify for some things that can be used for scripting. using vim.notify completly removes this ability to allow users to script around errors.

https://github.com/nvim-telescope/telescope.nvim/pull/2225 tries to resolve this and i plan to finish it.

we already merged this and had a whole discussion here on why we probably dont want to do it like that. https://github.com/nvim-telescope/telescope.nvim/pull/2181